src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/ComponentDiagnosticRazorIntegrationTest.cs

```diff
@@ -9,7 +9,7 @@ public class ComponentDiagnosticRazorIntegrationTest : RazorIntegrationTestBase
     {
         internal override string FileKind => FileKinds.Component;
 
-        [Fact(Skip = "https://github.com/aspnet/AspNetCore/issues/6217")]
```

Comment:
was there a test case that affects the error you removed?

What would the errors be in a case like:
```
<div>
  <foo>
 </bar>
</div>
```

I just want to make sure that we're not incredibly noisy with the errors.

Reply:
https://github.com/aspnet/AspNetCore-Tooling/pull/351/files#diff-79444749c3ee6081f2d9312fec5242d8R329 is the one that tests it. It doesn't verify what error it produces but makes sure we don't rewrite anything because it's an error case.

Reply:
I can add an integration test to verify what errors it produces. In you example, I expect it to error on `<foo>` and another error on `</bar>`.

Reply:
Spoke offline. @rynowak has no concerns as long as this doesn't create too much noise.